servers/audio/audio_rb_resampler.cpp

```diff
@@ -176,8 +176,8 @@ Error AudioRBResampler::setup(int p_channels, int p_src_mix_rate, int p_target_m
 		rb_bits = desired_rb_bits;
 		rb_len = (1 << rb_bits);
 		rb_mask = rb_len - 1;
-		rb = memnew_arr(float, rb_len *p_channels);
-		read_buf = memnew_arr(float, rb_len *p_channels);
+		rb = memnew_arr(float, rb_len *(size_t)p_channels);
+		read_buf = memnew_arr(float, rb_len *(size_t)p_channels);
```

Comment:
This code is a bit weird because this is definitely a multiplication, but clang-format insists that there shouldn't be a space after the asterisk (does it think this has something to do with pointers?)

Reply:
Could you try reversing the order of the operands, see if clang-format likes that better? I know it was like this before too, but it would read a bit more cleanly with a space in there.

Reply:
I swapped them around and clang-format still removes the space.

Reply:
Since we're calculating it twice, maybe do `const size_t arr_size = rb_len * p_channels;` beforehand.

There's also this one where it's bogus:
```
platform/javascript/audio_driver_javascript.cpp
122:    output_rb = memnew_arr(float, buffer_length *channel_count);
129:    input_rb = memnew_arr(float, buffer_length *channel_count);
```

Reply:
I don't think we should block this PR on fixing that issue but it does bother me.

Reply:
@ellenhp Nah, it's an easy fix on the same lines, so it makes sense to do it in this PR instead of waiting.

I've updated this PR to have these values be constants.

Reply:
Oh, I just meant the clang-format issue when I said that.

Reply:
The formatting has to be correct or else the CI checks will fail. The work-around of adding constants is a simple solution.

If you're talking about fixing the upstream bug in clang-format, yes I agree, that would take a very long time for the fix to arrive downstream even assuming we were able to get it fixed quickly so we shouldn't wait for that.